Update Connecting-with-a-VPC-Azure.mdx (#5336)
cube-js/cube.js: docs/content/Cube-Cloud/Configuration/VPC/Connecting-with-a-VPC-Azure.mdx

## Prerequisites

To proceed following information is required:

- **Virtual Network Name:** You can find it in the `Virtual Networks` section of
  the [Azure Portal][azure-console].
- **Tenant ID:** You can find it under `Azure Active Directory` > `Properties` >
  `Tenant ID` in the [Azure Portal][azure-console].

## Setup

> Unfortunately Azure cross-tenant peering suppose that you assign peering role to

Reviewer: ```suggestion
Unfortunately, Azure cross-tenant peering assumes that you assign the peering role to
```

the `user id/service principal` of the peering party.

> Below, list of the steps that would register `Cube Cloud tenant` at your organization and grant peering access to `Cube Cloud service principal`:

Reviewer: ```suggestion
Below is the list of the steps that would register `Cube Cloud tenant` at your organization and grant peering access to `Cube Cloud service principal`:
```

### Add Cube tenant to your organization

> At first, `Cube Cloud tenant` must be added to your organization. To do this,

Reviewer: ```suggestion
First, `Cube Cloud tenant` must be added to your organization. To do this,
```

open the [Azure Portal][azure-console] and go to `Azure Active Directory` > `External
Identities` > `Cross-tenant access settings` > `Organizational Settings` > `Add
Organization`.

For `Tenant ID`, enter `197e5263-87f4-4ce1-96c4-351b0c0c714a`. Make sure that
`B2B Collaboration` > `Inbound Access` > `Applications` is set (or inherited) so
that it `Allows access`

### Register Cube Cloud service principal at your organization

Using browser tab, where account that has permissions to register
`Enterprise applications` is logged, open the following url, replacing **__TenantID__** with your `Tenant ID`:
https://login.microsoftonline.com/__TenantID__/oauth2/authorize?client_id=0c5d0d4b-6cee-402e-9a08-e5b79f199481&response_type=code&redirect_uri=https%3A%2F%2Fwww.microsoft.com%2F

`Cube Cloud service principal` has following information, please double check that you see them exactly as they are on the dialog box that has poped up:
Client ID: `0c5d0d4b-6cee-402e-9a08-e5b79f199481`
Name: `cube-dedicated-infra-sp`

> When you made sure that all information is correct select `Consent on behalf of your organization` And click `Accept`

Reviewer: ```suggestion
After you made sure that all information is correct, select `Consent on behalf of your organization` And click `Accept`
```
